### PR TITLE
Auton intake

### PR DESCRIPTION
**Pull request by KyleQ1**

### CODE REVIEW

File: src/main/cpp/subsystems/Intake.cpp [hunk 1]
```diff
@@ -60,13 +60,9 @@ void Intake::RobotPeriodic() {
...
     if (m_flywheel.IsOn()) {
-        if (m_flywheel.IsReady()) {
-            SetArmMotor(ArmMotorDirection::kIntake);
-        } else {
-            SetArmMotor(ArmMotorDirection::kIdle);
-        }
+        SetArmMotor(ArmMotorDirection::kIntake);
     } else {
-        // Manual control
+        // Manual Control
```
Comment (calcmogul) [inline src/main/cpp/subsystems/Intake.cpp:65]: Don't capitalize this

File: src/main/cpp/subsystems/Intake.cpp [hunk 2]
```diff
@@ -81,8 +77,7 @@ void Intake::RobotPeriodic() {
     // Funnel logic
     if (m_flywheel.IsOn() || IsLowerSensorBlocked()) {
         if (m_flywheel.IsReady() ||
-            (!m_flywheel.IsOn() && !IsUpperSensorBlocked() &&
-             IsLowerSensorBlocked())) {
+            !IsUpperSensorBlocked() && IsLowerSensorBlocked()) {
```
Comment (calcmogul) [inline src/main/cpp/subsystems/Intake.cpp:80]: Were there any table changes needed for this?

**Review by calcmogul — CHANGES_REQUESTED** [review]
The pseudocode below the table needs to be updated as well.

File: src/main/cpp/subsystems/Intake.cpp (hunk 2 above)
Comment (KyleQ1) [reply src/main/cpp/subsystems/Intake.cpp]: Yeah, the table was changed to reflect that.

File: docs/intake-io-table.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Funnel logic:
 # if !!flywheelOn || !!lower:
 if flywheelOn || lower:
     # if (!flywheelOn && !flywheelReady && !upper && lower) || (flywheelReady):
```
Comment (calcmogul) [inline docs/intake-io-table.md:45]: You forgot to update this line.

**Pull request merged**